client/my-sites/plans/current-plan/jetpack-checklist/index.tsx

```diff
@@ -342,9 +346,13 @@ class JetpackChecklist extends PureComponent< Props & LocalizeProps > {
 }
 
 function mapStateToProps( state ) {
+	const OFFER_RESET_VIDEO_MINIMUM_JETPACK_VERSION = '8.9.2';
```

Comment:
I wonder if it would be better to have this constant defined in `lib/plans/constants` since we may have to use it for other features that we left behind in the Jetpack plugin. I'm totally fine with leaving it here and then moving into another file is need arises.

Reply:
Good call @rcanepa , similar fixes will probably be released in the plugin at the same time. I'll leave it as is for now but I'll soon have related PRs: I'll do the update then.